fluapigen-processor/src/main/java/io/toolisticon/fluapigen/processor/ModelInterfaceMethodParameter.java

```diff
@@ -218,6 +224,10 @@ else if (backBeanField.get().isArray()) {
         return "";
     }
 
+    private String generateSourceForIterableToList(String parameterName){
```

Comment:
It is strange that I have got the notification email upon your comment but I can not find it here. Sure it is my bad. Anyway, even if we return a List it is a Collection that can be used both for addAll and (now I have fixed) for new backed bean instance creation.

Reply:
I had to remove the cast here, the parameter will always be of type Iterable including the correct Generic type declaration

Reply:
Sure it was a mistake from my side. This specific cast is applicable only for the CUTE usage but Fluent API Generator is a generic tool. Thanks for catching.